cmd/run.go

```diff
@@ -422,9 +423,16 @@ func historyPath() string {
 }
 
 func loadCertificate(tlsCertFile, tlsPrivateKeyFile string) (*tls.Certificate, error) {
-
 	if tlsCertFile != "" && tlsPrivateKeyFile != "" {
-		cert, err := tls.LoadX509KeyPair(tlsCertFile, tlsPrivateKeyFile)
+		tlsCertFilePath, err := fileurl.Clean(tlsCertFile)
+		if err != nil {
+			return nil, err
```

Comment:
Since we're loading multiple files here, it'd help to say which file failed in the returned error. E.g.

```suggestion
			return nil, fmt.Errorf("invalid certificate file path: %w", err)
```

Same for the private key file path down below.

Reply:
Fixed!